packages/wallets/src/features/cashier/modules/DepositCrypto/components/DepositCryptoAddress/__tests__/DepositCryptoAddress.spec.tsx

```diff
@@ -23,33 +17,26 @@ describe('DepositCryptoAddress', () => {
         HTMLCanvasElement.prototype.getContext = jest.fn();
         (useDevice as jest.Mock).mockReturnValue({ isMobile: false });
         (useHover as jest.Mock).mockReturnValue(false);
-        (useAuthorize as jest.Mock).mockReturnValue({ isSuccess: true });
-        (useDepositCryptoAddress as jest.Mock).mockReturnValue({
-            data: '1A1zP1eP5QGefi2DMPTfTL5SLmv7DivfNa',
-            isLoading: false,
-            mutate: jest.fn(),
-        });
     });
     it('should show loader when crypto address not loaded yet', () => {
-        (useDepositCryptoAddress as jest.Mock).mockReturnValue({ data: undefined, isLoading: true, mutate: jest.fn() });
-        render(<DepositCryptoAddress />);
+        render(<DepositCryptoAddress depositCryptoAddress={undefined} isLoading={true} />);
```

Comment:
@lubega-deriv 
if we remove the loader as per comment above we will also remove it from the test, but just FYI we can isLoading and it will be considered as a true

Reply:
Updated